src/views/Home.jsx

```diff
@@ -17,15 +20,55 @@ export function Home({ createToken, setListToken }) {
 		} else {
 			listId = null;
 			setListToken(listId);
-			setMessage('Your shopping list was not created. Please try again. ');
+			setCreateListMessage(
+				'Your shopping list was not created. Please try again. ',
+			);
 		}
 	}
 
+	async function handleJoinClick(e) {
+		e.preventDefault();
+
+		if (tokenInput) {
+			let listId = tokenInput;
```

Comment:
**Nitpick:** Is the `listId` variable redundant here? Could you simply use the `tokenInput` variable in its place for the same outcome?

Reply:
Good catch. This has been updated :)